content/source/docs/cloud/users-teams-organizations/single-sign-on/okta.html.md

```diff
@@ -48,3 +50,25 @@ Be sure to copy the metadata URL (from the final step of configuring Okta) befor
 6. Your Okta SSO configuration is complete and ready to [use](../single-sign-on.html#using-sso).
 
     ![sso-settings](../images/sso/settings-okta.png)
+
+## Configuration - Team Mapping (Okta)
+1. Complete all steps outlined in the [Configuration (Okta)](./okta.html#configuration-okta-) section above, and take note of the default team mapping behavior as described [here](../single-sign-on.html#managing-team-membership-through-sso).
+
+2. Edit your Terraform Cloud Okta Application and complete the following steps:
+* Expand the `Attributes` section of the Application configuration (under the `Sign On` tab):
+
+    ![sso-okta-attribute-settings](../images/sso/okta-attribute-settings.png)
```

Comment:
For accessibility reasons, images must include descriptions. Users with low vision rely on these. 

The description goes in the square brackets here, where you currently have the filename repeated. 

When writing a description, focus on what's most important to a low-vision user during this workflow. In this image, it would probably be the fact that the button is next to the text "Attributes (optional)". For the others it'll be something different.

Reply:
Thank you!  I've added more meaningful descriptions to these that should make them more useful.